capstone/src/main/java/com/google/sps/Objects/User.java

```diff
@@ -4,24 +4,26 @@
 
 public final class User {
 
-  private final int userId;
+  private final String userId;
   private final String name;
+  private final String firstname;
   private final String email;
   private final ArrayList<Badge> badges;
-  private final ArrayList<Group> groups;
+  private final ArrayList<String> groups;
   private final ArrayList<String> interests;
 
-  public User(int userId, String name, String email, ArrayList<Badge> badges,
-      ArrayList<Group> groups, ArrayList<String> interests) {
+  public User(String userId, String firstname, String lastname, String email, 
```

Comment:
we could add get/set methods for first name, last name, phone number, interests

Reply:
I would also recommend camel casing `firstName`, `lastName`